core/src/syn/parser/stmt/define.rs

```diff
@@ -1212,6 +1216,122 @@ impl Parser<'_> {
 		Ok(res)
 	}
 
+	pub async fn parse_define_config(
+		&mut self,
+		ctx: &mut Stk,
+	) -> ParseResult<DefineConfigStatement> {
+		let (if_not_exists, overwrite) = if self.eat(t!("IF")) {
+			expected!(self, t!("NOT"));
+			expected!(self, t!("EXISTS"));
+			(true, false)
+		} else if self.eat(t!("OVERWRITE")) {
+			(false, true)
+		} else {
+			(false, false)
+		};
+
+		let next = self.next();
+		let inner = match next.kind {
+			t!("GRAPHQL") => {
+				self.pop_peek();
+				self.parse_graphql_config(ctx).await.map(ConfigInner::GraphQL)?
+			}
+			_ => unexpected!(self, next, "a type of config"),
+		};
```

Comment:
You call next here consuming a token and then call pop_peek which would pop a peeked token. However the function never peeks a token so this will either pop a look ahead token which is wrong because that token was never used or it will do nothing.